vm/vm.go

```diff
@@ -150,6 +128,40 @@ func (vm *VM) newThread() (t Thread) {
 	return
 }
 
+// vm.assignLibPath looks up and assigns vm.libPath
+func (vm *VM) assignLibPath() (err error) {
+	if DefaultLibPath != "" {
+		vm.libPath = DefaultLibPath
+		return
+	}
+
+	gobyRoot := os.Getenv("GOBY_ROOT")
+
+	if len(gobyRoot) == 0 {
+		// fallback to homebrew's path
+		gobyRoot = fmt.Sprintf("/usr/local/Cellar/goby/%s", Version)
+
+		_, err := os.Stat(gobyRoot)
```

Comment:
I'm not sure what's the standard here, but I was wondering if the standard Go pattern is:

```golang
if _, err := os.Stat(gobyRoot); err != nil {
  // ...
}
```